weasyl/login.py

```diff
@@ -317,7 +315,7 @@ def verify(token, ip_address=None):
         # Create login record
         userid = db.scalar(lo.insert().returning(lo.c.userid), {
             "login_name": d.get_sysname(query.username),
-            "last_login": arrow.now(),
+            "last_login": datetime.now(tz=pytz.UTC),
```

Comment:
Does `sqlalchemy.func.now()` work here?

Reply:
It needs to be changed around a bit to make it work. Pushed a change with it, let me know if you like it better, or if I should revert it :)